Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -323,3 +323,31 @@ private extension WorldScaleGeoTrackingSceneView {
         )
     }
 }
+
+public extension WorldScaleGeoTrackingSceneView {
+    /// Determines the scene point for the given screen point.
+    /// - Parameter screenPoint: The point in screen's coordinate space.
+    /// - Returns: The scene point corresponding to screen point.
+    func arScreenToLocation(screenPoint: CGPoint) -> Point? {
+        // Use the `raycast` method to get the matrix of `screenPoint`.
+        guard let localOffsetMatrix = arViewProxy.raycast(from: screenPoint) else { return nil }
+        
+        let translationFactor = cameraController.translationFactor
```

Comment:
For world scale AR, the translation factor should always be 1. I'll change it in the next commit.